src/Sulu/Bundle/AdminBundle/Resources/js/services/Requester/RequestPromise.js

```diff
@@ -0,0 +1,28 @@
+//@flow
+
+export default class RequestPromise<T> extends Promise<T> {
+    abortController: ?AbortController;
+
+    setAbortController(abortController: ?AbortController) {
+        this.abortController = abortController;
+    }
+
+    abort() {
+        if (!this.abortController) {
+            throw new Error('A request can only be aborted if the setAbortController function was called.');
+        }
+        this.abortController.abort();
+    }
+
+    then(onFulfilled: ?(*) => Promise<*> | *, onRejected: ?(*) => Promise<*> | *): RequestPromise<*> {
+        const requestPromise: RequestPromise<*> = ((super.then(onFulfilled, onRejected): any): RequestPromise<*>);
+        requestPromise.setAbortController(this.abortController);
+
+        return requestPromise;
+    }
+
+    catch(onReject: ?(*) => Promise<*> | *): RequestPromise<*> {
+        // $FlowFixMe
```

Comment:
Removed the `$FlowFixMe` and set the AbortController. Strictly speaking the latter would probably not be necessary, because you can't cancel something that has already failed... Added it anyway just to be sure, maybe there is some kind of edge case.